### PR TITLE
Add graphql query to add document to a component

### PR DESCRIPTION
**Pull request by aliesesay**
Hello team,

As discussed with Josh Campbell, I'm raising this PR to add GraphQL query examples to add a document to a Compass component. 

References:
https://developer.atlassian.com/cloud/compass/graphql/#mutations_addDocument)
https://developer.atlassian.com/cloud/compass/graphql/#queries_documentationCategories

### CODE REVIEW

File: snippets/graphql/add-document-to-component/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    }
+  }
+}
+
+##Query Variables
+
+```graphql
+{
+  "cloudId": "YOUR-CLOUD-ID",
+  "first": 10,
```
Comment (ebudtke-atl) [inline snippets/graphql/add-document-to-component/README.md:46]: Nit: you could limit this to just 4 as right now the categories are static so you won't ever get back more than that for a given site.

File: snippets/graphql/add-document-to-component/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+
+From the above result, you will get the Document Category ID in the ARI format for the following types:
+
+Discover
+Contributor
+Maintainer
+Other
+
+Step 2: Use the following query to add a document to your Compass
```
Comment (ebudtke-atl) [inline snippets/graphql/add-document-to-component/README.md:58]: Nit: Should be "Use the following _mutation_ ... "

File: snippets/graphql/add-document-to-component/README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    }
+  }
+}
+
+##QUERY VARIABLES
+
+```graphql
+{
+  "input": {
+    "title": "title",
```
Comment (ebudtke-atl) [inline snippets/graphql/add-document-to-component/README.md:86]: Would it be worth calling out (in a comment or further above in the readme) that title is optional and you can just supply a url?

File: snippets/graphql/add-document-to-component/README.md (hunk 1 above)
Comment (aliesesay) [reply snippets/graphql/add-document-to-component/README.md:46]: ok

**Review by ebudtke-atl — APPROVED** [review]

**Comment by aliesesay** [comment]
CLA signed

**Pull request merged**